doc/endpoints/track.md

```diff
@@ -31,7 +30,7 @@ This method is used to get information about multiple tracks in the Spotify cata
 #### Parameters
```

Comment:
Mention the endpoint and add the link to the spotify page (just we did for the first method)